python/doc/examples/meta_modeling/general_purpose_metamodels/plot_create_linear_least_squares_model.py

```diff
@@ -9,7 +9,7 @@
 # We consider the function :math:`h : \Rset^2 \rightarrow \Rset^2` is defined by:
 #
 # .. math::
-#    h(x_1, x_2) = \left(\cos(x_1 + x_2), (x_2 + 1) e^{x_1 - 2 x_2} \right)
+#    \vect{h}(x_1, x_2) = \left(\cos(x_1 + x_2), (x_2 + 1) e^{x_1 - 2 x_2} \right)
```

Comment:
In mathematics, there are only column vectors, not row vectors. This is because vectors are associated with an algebra. Hence, a ~~row vector~~ is actually a matrix, which is not the expected meaning here.

In order to introduce a transpose symbol, the "Tr" macro uses the "top" symbol (although it might not be the best option, see [stackexchange](https://tex.stackexchange.com/questions/30619/what-is-the-best-symbol-for-vector-matrix-transpose)):
```suggestion
#    \vect{h}(x_1, x_2) = \Tr{\left(\cos(x_1 + x_2), (x_2 + 1) e^{x_1 - 2 x_2} \right)}
```
Please check that the rendering corresponds to the expected equation.